guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -91,8 +85,10 @@ ifdef::satellite[]
 endif::[]
 . Upgrade {ProjectServer} and all {SmartProxyServer}s to {Project} {ProjectVersion}.
```

Comment:
This is now the only step remaining in the "high level overview" on non-Katello. It feels really weird but I'm not sure what to do with it.

I've been thinking about splitting it up:
```suggestion
. Upgrade {ProjectServer} to {ProjectVersion}.
. Upgrade all {SmartProxyServer}s to {ProjectVersion}.
```

This would be correct in all scenarios and matches the headers. On the other hand, perhaps it's better to just skip the whole section to keep things short.

Reply:
I think splitting it up is the way to go. Users can then still do both procedures sequentially.